lib/CheckToggle/index.js

```diff
@@ -88,6 +88,7 @@ class CheckToggle extends Component {
         {labelLeft && <p className={wrapperLabelClasses}>{labelLeft}</p>}
         <span className="toggle-wrapper__inner">
           <input
+            data-testid={id}
```

Comment:
Do we need this? Cant we just grab this by the label text?

Reply:
Good point, we should be able to!

Reply:
> Do we need this? Cant we just grab this by the label text?

To follow up on this, the `CheckToggle` component basically contains debt that means we cannot target it `byLabelText`.

The current `CheckToggle` component uses a `<label>` element as the actual switch (and hides the native input), and uses `<p>` tags for the labels.

I took a look at the refactor and I think it is going to go our the scope of this PR. One for the future though under lesser time pressures 👍